recipes/emptydrops/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "emptydrops" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 16552739852ccb39347f6f1ecddbe10abf4b14ae6b59d5c7b1d4091ad9806cb8
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install --no-deps --ignore-installed -vv ."
+
+requirements:
+  host:
+    - python >=3
+  run:
+    - h5py
+    - lz4
+    - pandas
+    - scipy
+
+test:
+  imports:
+    - emptydrops
+    - emptydrops.matrix
+
+about:
+  home: https://pypi.org/project/emptydrops/
+  dev_url: https://github.com/nh3/emptydrops
+  license: MIT
```

Comment:
can you please add a license_file

Reply:
You mean adding this file (https://github.com/nh3/emptydrops/blob/master/LICENSE)? Surely very ignorant, but I haven't seen a sub-directory under `recipe/` having a license_file.

Reply:
Normally you are shipping this file in your pypi package. In this case you just add `license_file: LICENSE`

Reply:
Ah, I see. Thanks!